app/controllers/state_file/questions/az_senior_dependents_controller.rb

```diff
@@ -0,0 +1,17 @@
+module StateFile
+  module Questions
+    class AzSeniorDependentsController < QuestionsController
+      def self.show?(intake)
+        !intake.nil? && intake.dependents.select(&:ask_senior_questions?).size.positive?
```

Comment:
Not a huge deal but looking through the existing `show?` implementations in the app, none of them seem to guard against the presence of `intake` -- I think typically they're invoked with a real one or they're not invoked at all

The state file flow is still immature, the other flows typically have a `before_action` installed on them after login that redirects out of the flow if the sessioned intake is nil. Maybe that has something to do with it?

would be curious to know what starts blowing up if the nil-guard is removed

Reply:
there is a flow_controller_spec or something that blew up without this guard. it was outside the bin/test state_file target so it blew up after I pushed my original change and then I added this. 

I didn't look real closely at whether the test could be changed instead, could look into that

Reply:
I think one of Ben's to-be-merged PRs has some changes that might make the flow controller spec thing work better; anyway it's fine to keep in the guard for now maybe it's something we'll notice and remove later.